src/Frontend/Components/AttributionCountPerSourcePerLicenseTable/AttributionCountPerSourcePerLicenseTable.tsx

```diff
@@ -40,34 +57,163 @@ export const AttributionCountPerSourcePerLicenseTable: React.FC<
     .reduce((partialSum, num) => partialSum + num, 0)
     .toString();
 
-  const footerRow = [FOOTER_TITLE]
-    .concat(totalNumberOfAttributionsPerSource)
-    .concat(totalNumberOfAttributions);
-  const headerRow = [LICENSE_COLUMN_NAME_IN_TABLE]
-    .concat(sourceNames)
-    .concat(TOTAL_SOURCES_TITLE)
-    .map(
-      (sourceName) => sourceName.charAt(0).toUpperCase() + sourceName.slice(1),
-    );
+  const footerRow = [
+    text.attributionCountPerSourcePerLicenseTable.footerTitle,
+    '',
+    '',
+    ...totalNumberOfAttributionsPerSource,
+    totalNumberOfAttributions,
+  ];
+
+  const headerRow = [
+    text.attributionCountPerSourcePerLicenseTable.columnNames.licenseName,
+    text.attributionCountPerSourcePerLicenseTable.columnNames.criticality,
+    text.attributionCountPerSourcePerLicenseTable.columnNames.classification,
+    ...sourceNames,
+    text.attributionCountPerSourcePerLicenseTable.columnNames.totalSources,
+  ].map(
+    (sourceName) => sourceName.charAt(0).toUpperCase() + sourceName.slice(1),
+  );
 
   Object.entries(
     props.licenseCounts.attributionCountPerSourcePerLicense,
   ).forEach(
     ([licenseName, value]) =>
-      (value[TOTAL_SOURCES_TITLE] =
+      (value[TOTAL_SOURCE_NAME] =
         props.licenseCounts.totalAttributionsPerLicense[licenseName]),
   );
 
+  const tableHead = (
+    <MuiTableHead>
+      <MuiTableRow>
+        {headerRow.map((columnHeader, columnIndex) => (
+          <MuiTableCell
+            sx={tableClasses.head}
+            key={columnIndex}
+            align={columnIndex === 0 ? 'left' : 'center'}
+          >
+            {columnHeader}
+          </MuiTableCell>
+        ))}
+      </MuiTableRow>
+    </MuiTableHead>
+  );
+
+  const buildTableRow = (licenseName: string, rowIndex: number) => {
+    const licenseNameCell = (
+      <MuiTableCell
+        sx={{
+          ...tableClasses.body,
+        }}
+        key={0}
```

Comment:
a constant key is unnecessary
```suggestion
        sx={tableClasses.body}
```